test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] This file is ignored by `.gitignore`. Kindly Fix the path for this file in `.gitignore`.